tests/unit/data/endpoints/valid-rules/aws-credential-scope.json

```diff
@@ -0,0 +1,95 @@
+{
```

Comment:
Has there been any discussion about making this a more comprehensive ruleset so we don't have the same boiler plate each time?

Reply:
I don't think so. I did write the last few by hand whereas the existing files were all shipped with the endpoints refactor last year. Since they're all pretty specific I elected to keep them separate. If we want to combine them I think it'd be better to do that as a followup PR.